jobs/gorouter/spec

```diff
@@ -266,12 +266,41 @@ properties:
       description: "Enforce strict validation of a route service signature"
       default: false
   router.max_header_kb:
+    description: |
+        Deprecated, use router.max_request_header_kb instead which is equivalent to this option.
+    default: 1024 # 1Mb
+  router.max_request_header_kb:
     description: |
         This value controls the maximum number of bytes (in KB) the gorouter will read
```

Comment:
You have two descriptions here. I think you meant to combine them?

Reply:
The one for the existing property `max_header_kb` got cut to the deprecation notice and I've added `max_request_header_kb` with all the details. The diff is a bit weird because it noticed that I kept most of the existing description from the previous option.